app-admin/pass-otp/pass-otp-1.1.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A pass extension for managing one-time-password (OTP) tokens."
+HOMEPAGE="https://github.com/tadfisher/pass-otp"
+SRC_URI="https://github.com/tadfisher/pass-otp/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qrcode test"
+
+DEPEND="test? ( dev-tcltk/expect:* )"
+
+RDEPEND=">=app-admin/pass-1.7
+	qrcode? ( media-gfx/qrencode )
+	sys-auth/oath-toolkit"
+
+src_compile() {
+	:
+}
+
+pkg_postinst() {
+	elog "'PASSWORD_STORE_ENABLE_EXTENSIONS=true' is needed to run 'pass otp'."
```

Comment:
Please elaborate on this. If someone knows little about pass, this is going to be one big 'whaaaaaat? where?'